### ISSUE TITLE
upgrade to 0.3.0 still need kube-delivery binary

### ISSUE THREAD

**Issue by AlexChenSkyBlue** [kubeflow/mpi-operator#433]
/bin/sh: 1: /opt/kube/kubectl: not found
/bin/sh: 1: /opt/kube/kubectl: not found

**Comment by AlexChenSkyBlue** [comment]
> isn't kubectl-delivery in a separate image? https://hub.docker.com/r/mpioperator/kubectl-delivery
> 
> cc @terrytangyuan

Yes. but 0.3.0 state already dropped kubectl-delivery requirement

**Comment by alculquicondor** [comment]
Please provide detailed steps to repro

**Comment by tenzen-y** [comment]
The kubectl-delivery was removed in the MPIJob v2, and MPIJob v1 is no longer maintained in this repo.
/close
